src/app/features/reporting/report-config.ts

```diff
@@ -69,13 +100,39 @@ export interface ExportingReport extends ReportConfig {
  */
 export interface SqlReport extends ReportConfig {
   mode: "sql";
+
   /**
-   * a valid SQL SELECT statements, can contain "?" placeholder for arguments
+   * version of the ReportConfiguration, currently 1 or 2
+   */
+  version: number;
+
+  /**
+   * a valid SQL SELECT statements, can contain "?" placeholder for arguments (only v1)
    */
   aggregationDefinition: string;
 
   /**
-   * a list of arguments, passed into the sql statement
+   * a list of arguments, passed into the sql statement (only v1)
    */
   neededArgs: string[];
+
+  /**
+   *  (sql v2 only) transformations that are applied to input variables (e.g. startDate, endDate)
+   *  example: {startDate: ["SQL_FROM_DATE"], endDate: ["SQL_TO_DATE"]}
```

Comment:
maybe we remove the detailed code comment in the class (or here in the interface; which one is imported in other services?) and replace it with a "see SqlReport interface", so that we do not have to keep the comments updated in two places?